### DISCUSSION TITLE
Best Practices for Working with KSADS Codes

### DISCUSSION THREAD

**Discussion by vicgwilli** [now-i-know-my-abcd/docs#42] [Stats & Code Help]
Are there best practices for how to use KSADS codes 555, 777, 888, 999? For instance, I am looking to simplify the data and potentially recode these. Are the 888s akin to a '0' or 'no' to a given item because, presumably, the participant didn't get asked that item because they didn't meet screener criteria for further follow-up questions about that item and thus would likely not meet? Whereas 555 would be better coded as 'NA'?

I know we've [already got some answers ](https://github.com/now-i-know-my-abcd/docs/discussions/10#discussion-4273904) for what these codes stand for, I'm just wondering about recommendations for simplifying/consolidating them for analyses...

**Comment by niamhmacsweeney** [comment]
Sorry for the delayed reply Victoria! I'm not sure about best practices for KSADS scoring. When I've worked with KSADS data in the past in ABCD, I've recoded "999" as "NA". I had a look on the data dictionary for release 4.0 but I couldn't see the different codes (e.g., 555, 777) listed for the full KSADS (field name: abcd_ksad01). Is there a reason you are working with the full KSADS rather than the individual symptom scales (e.g., social_anxiety_disorder_p01)?
